src/styles.css

```diff
@@ -0,0 +1,158 @@
+body {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  background-color: #6d6970;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: black;
+  position: fixed;
+  width: 100%;
+}
+
+#logo {
+  display: flex;
+}
+
+#logo1 {
+  height: 5rem;
+}
+
+#title {
+  color: #967843;
+}
+
+#header-nav {
+  display: flex;
+  justify-self: end;
+}
+
+#nav-list {
+  display: flex;
+  list-style: none;
+  justify-content: space-around;
+  gap: 20px;
+  font-size: 1.5rem;
+  cursor: pointer;
+  color: #967843;
+  margin-right: 45px;
+}
+
+footer {
+  text-align: start;
+  background-color: black;
+  margin-top: 10px;
+}
+
```

Comment:
- [ ]  `Home page footer` should be similar to the given [mockup](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md)

Your layout  | Wireframe layout
------------- | -------------
<img width="1397" alt="Screenshot 2022-05-20 at 18 07 39" src="https://user-images.githubusercontent.com/77038610/169545573-38d8add1-0c29-40ef-b695-b08b7de7789e.png"> | <img width="538" alt="Screenshot 2022-05-20 at 18 07 51" src="https://user-images.githubusercontent.com/77038610/169545602-5dbc5fce-e6bf-4781-b1ae-8caca451067b.png">

Kindly align footer to the left